cost-analyzer/values.yaml

```diff
@@ -2235,6 +2235,46 @@ kubecostDeployment:
   labels: {}
   annotations: {}
 
+## Kubecost Forecasting forecasts future cost patterns based on historical
+## patterns observed by Kubecost.
+forecasting:
+  enabled: false
+
+  # fullImageName overrides the default image construction logic. The exact
+  # image provided (registry, image, tag) will be used for the forecasting
+  # container.
+  # Example: fullImageName: gcr.io/kubecost1/forecasting:v0.0.1
+  fullImageName: gcr.io/kubecost1/kubecost-modeling:6ff46def0575545eb37c5dae179e5035a1de279f
```

Comment:
We plan to release forecasting like Cluster Controller and Network Costs, with a fixed image in the Helm chart (instead of the branching/tagging used for higher-traffic repos like KCM). The particular images being set here are development images that are very close to the final release state.